resources/sass/_base.scss

```diff
@@ -84,3 +88,20 @@ p {
 ::selection {
     @apply bg-blue-500 text-white;
 }
+
+::-webkit-scrollbar {
```

Comment:
I think that the scrollbar needs yo change only on the sidebar so we can do something like that
```
.scrollbar::-webkit-scrollbar {
    width: 8px;
}

/* Track */
.scrollbar::-webkit-scrollbar-track {
    @apply bg-gray-300 bg-opacity-30;
}

/* Handle */
.scrollbar::-webkit-scrollbar-thumb {
    @apply bg-gray-400 bg-opacity-30;
}

/* Handle on hover */
.scrollbar::-webkit-scrollbar-thumb:hover {
    @apply bg-gray-500;
}
```